{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/core/signals.py

```diff
@@ -1,11 +1,37 @@
+import logging
+
+{%- if cookiecutter.include_mobile == 'y' %}
+from django.conf import settings
+{%- endif %}
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 from rest_framework.authtoken.models import Token
 
+from {{ cookiecutter.project_slug }}.core.dispatch import new_reset_password_code_created_ds
 from {{ cookiecutter.project_slug }}.core.models import User
+from {{ cookiecutter.project_slug }}.utils.emails import send_html_email
+
+logger = logging.getLogger(__name__)
```

Comment:
Need to check but, if not mobile, will this give a linting error due to unused variable?
If so, we might also want to set it conditionally